mongodb-storage/src/main/java/io/lumeer/mongodb/MongoDbStorageDialect.java

```diff
@@ -67,6 +69,10 @@ public DataDocument addRecentlyUsedDocumentQuery(final String metadataCollection
                               .append("$slice", listSize))));
    }
 
+   public String sortStringForSearch(final String field, final int order) {
+      return MongoUtils.convertBsonToJson(Sorts.descending(field));
```

Comment:
I'm sorry I didn't check it. It is forgotten method and I have already removed it.